ntoskrnl/ke/amd64/traphandler.c

```diff
@@ -0,0 +1,242 @@
+/*
+ * PROJECT:         ReactOS Kernel
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * PURPOSE:         x64 trap handlers
+ * PROGRAMMERS:     Timo Kreuzer (timo.kreuzer@reactos.org)
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include <ntoskrnl.h>
+#include <fltkernel.h>
```

Comment:
For what purpose is the filter header necessary?

Reply:
It was probably needed for `Add2Ptr` at some point, but it's no longer needed. Removed it in both files.